apps/web/src/services/asset-inventory/components/ServiceAccountBaseInformation.vue

```diff
@@ -129,7 +130,7 @@ export default defineComponent<Props>({
                 state.providerData = await SpaceConnector.clientV2.identity.provider.get({
                     domain_id: state.domainId, // TODO: remove domain_id after backend is ready
                     provider: props.provider,
-                });
+                } as ProviderGetRequestParams);
```

Comment:
suggestion: I recommend to use type declaration instead of type assertion like below:

```typescript
 state.providerData = await SpaceConnector.clientV2.identity.provider.get<ProviderGetRequestParams>({
                    domain_id: state.domainId, // TODO: remove domain_id after backend is ready
                    provider: props.provider,
                });
```